platform/conf/conf.go

```diff
@@ -405,6 +421,13 @@ func (c *Conf) copyKeysIgnitionV21(keys []*agent.Key) {
 	for _, key := range keys {
 		keyObjs = append(keyObjs, v21types.SSHAuthorizedKey(key.String()))
 	}
+	for i := range c.ignitionV21.Passwd.Users {
```

Comment:
note, in all these loops we can use `for _, user :=` ... The usual iter-value-reused thing doesn't apply to us here because of the early return.

I think it is a bit more readable to do that in all these loops

Reply:
That's what I had first and it was broken. Those are slices of structs, not slices of references to structs, and we're modifying one of them in place.

Reply:
Oh, whoops. Missed that '&' there, you're right.